src/validators/location.js

```diff
@@ -8,7 +8,7 @@ export const isInternational = location => {
   )
 }
 
-export const countryString = (country) => {
+export const countryString = country => {
```

Comment:
Actually, I would leave the `()` here; eslint convention has parens surrounding the arguments of any function that either

a). has multiple arguments
b). has  code other than a statement to be returned. in other words, a function with `{}`

Reply:
This change is coming from `prettier` - I've been running it based off of https://github.com/18F/e-QIP-prototype#formatting. We have a config file setup for `prettier`, so are parens with single arguments something we want to permit?

Reply:
Doesn't matter to me! The typical eslint default is the one I've described, that's the one I've used most commonly while at 18F, but if we already have something else defined, we can go with that. 

Does prettier run automatically? It didn't realize there was  any style linting at all, since the files in the project are so inconsistent. We may want to have it automatically run when we run our tests.

Reply:
Added a new issue to have Prettier run automatically https://github.com/18F/e-QIP-prototype/issues/1076 and will address in a separate PR if that's 👍